src/main/java/com/quirko/app/GameController.java

```diff
@@ -63,9 +76,173 @@ public ViewData onRotateEvent(MoveEvent event) {
     }
 
 
+    public static void saveGame(String fileName){
+        
+        String boardMatrix = "", currentShape = "", nextShape = "", posX = "" + getPosX(), posY = "" +  getPosY(), score = "" +  getScore();

```

Comment:
Bu implementasyonu yeni bir klass içerisine taşı. Buraya eklediğin kod GameController classını gereğinden fazla şişiriyor ve okunaklılığını azaltıyor.